include/fast_float/parse_number.h

```diff
@@ -91,7 +91,12 @@ from_chars_result from_chars(const char *first, const char *last,
   }
   answer.ec = std::errc(); // be optimistic
   answer.ptr = pns.lastmatch;
-
+  // Special fast path for integers.
+  if((pns.exponent == 0) && !pns.too_many_digits) {
```

Comment:
@lemire maybe there is sense to use "unlikely" here ?

Reply:
Maybe but most integers will qualify for Clinger’s fast path which comes next, and it is very fast... so, from first principles, we know the gains from this path are modest.

I am not saying it can’t be made to work... but it just seems that it is not worth it. I’d love to be proven wrong!